src/main/java/carpetextra/CarpetExtraSettings.java

```diff
@@ -100,77 +99,78 @@ public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentR
     )
     public static boolean dispensersFillMinecarts = false;
     
-    //@Rule(desc = "Wet sponges dry in the nether dimension", category = {EXTRA, FEATURE})
+    //@Rule(desc = "Wet sponges dry in the nether dimension.", category = {EXTRA, FEATURE})
     //public static boolean spongesDryInTheNether = false;
 
     @Rule(
-            desc = "Clerics can warm nether farts",
-            extra = {"This will also make them pick up wart items","As well as pathfind to soulsand"},
+            desc = "Clerics can warm nether farts.",
+            extra = {"This will also allow them to pick up wart items, as well as pathfind to soulsand."},
             category = {EXTRA, FEATURE}
     )
     public static boolean clericsFarmWarts = false;
     
-    @Rule(desc = "Multiple ice crushed by falling anvils make packed ice", category = {EXTRA, EXPERIMENTAL, FEATURE})
+    @Rule(desc = "Multiple ice crushed by falling anvils make packed ice.", category = {EXTRA, EXPERIMENTAL, FEATURE})
     public static boolean renewablePackedIce = false;
     
     @Rule(
-            desc = "Dispensers can play records if there's a jukebox in front of them",
-            extra = "If record already exists in the jukebox, it gets placed back in the dispenser",
+            desc = "Dispensers can play records if there's a jukebox in front of them.",
+            extra = "If a record already exists in the jukebox, it gets placed back in the dispenser.",
             category = {EXTRA, FEATURE, DISPENSER}
     )
     public static boolean dispensersPlayRecords = false;
 
     @Rule(
-            desc = "Client can provide alternative block placement",
+            desc = "Client can provide alternative block placement.",
             category = {EXTRA, SURVIVAL}
     )
     public static boolean accurateBlockPlacement = false;
 
     @Rule(
-            desc = "Dispensers can toggle with a stick things like redstone, noteblocks, comparator, daylight detector, repeaters",
+            desc = "Dispensers containing a stick can toggle things.",
+            extra = {"Works with buttons, redstone, noteblocks, comparators, repeaters", "and daylight detectors."},
```

Comment:
dispensersToggleThings not only works with these listed blocks. maybe adding a `etc.` or something will be better

Reply:
Oh, i thought these where the only blocks...i will add the etc directly as a commit, im sure gnembon wont mind :p

Reply:
Resolved in https://github.com/gnembon/carpet-extra/commit/f87c015e0b6165ef9e5e6a024bc32c7f5d255ee8